saleor/dashboard/customer/views.py

```diff
@@ -28,6 +28,8 @@ def customer_list(request):
     customers, paginator = paginate(customers, 30, request.GET.get('page'))
     ctx = {'customers': customers, 'form': form, 'title': title,
            'paginator': paginator}
+    if not request.GET:
+        ctx.update({'default_pagination_params': {'order_status': 'on'}})
```

Comment:
Why do we need this? Shouldn't the view behave the same no matter what page you're viewing? By default there are no GET params yet the view is filtered.